src/tests/run_calc_for_tests.ts

```diff
@@ -28,13 +28,15 @@ Object.defineProperty(window, 'matchMedia', {
 });
 
 HTMLCanvasElement.prototype.getContext = () => ({
-  scale: () => {}
+  scale: () => {
+    // Mock function!
+  }
 } as any)
 HTMLCanvasElement.prototype.toDataURL = () => ""
 HTMLMediaElement.prototype.play = async () => {
   await new Promise<void>((resolve) => resolve())
 }
-console.debug = () => {}
+// console.debug = () => {}
```

Comment:
```suggestion
console.debug = () => {}
```
I think this is intended; test results shouldn't include `console.debug` logs.